lib/acmesmith/post_issueing_hooks/sendmail.rb

```diff
@@ -0,0 +1,23 @@
+require 'acmesmith/post_issueing_hooks/base'
+
+module Acmesmith
+  module PostIssueingHooks
+    class Sendmail < Base
```

Comment:
This doesn't do anything (looks like just for concept). Please remove this file because I understand the concept of this PR :wink: